common/static/common/templates/discussion/thread.underscore

```diff
@@ -1,29 +1,28 @@
 <article class="discussion-article" data-id="<%- id %>" tabindex="-1">
     <div class="thread-wrapper">
-        <div class="forum-thread-main-wrapper">
+        <div class="thread-main-wrapper">
             <div class="thread-content-wrapper"></div>
             <div class="post-extended-content">
                 <ol class="responses js-marked-answer-list"></ol>
             </div>
         </div>
-        <div class="post-extended-content">
-            <div class="response-count"/>
+        <div class="post-extended-content thread-responses-wrapper">
             <% if (!readOnly) { %>
                 <div class="add-response">
-                    <button class="button add-response-btn">
-                        <i class="icon fa fa-reply"></i>
-                        <span class="add-response-btn-text"><%- gettext("Add a Response") %></span>
+                    <button class="btn-brand btn-small add-response-btn">
+                        <%- gettext("Add a Response") %>
                     </button>
                 </div>
             <% } %>
+            <div class="response-count"/>
             <ol class="responses js-response-list"/>
             <div class="response-pagination"/>
             <div class="post-status-closed bottom-post-status" style="display: none">
               <%- gettext("This thread is closed.") %>
             </div>
             <% if (can_create_comment && !readOnly) { %>
             <form class="discussion-reply-new" data-id="<%- id %>">
-                <h4><%- gettext("Post a response:") %></h4>
+                <h4><%- gettext("Add a response:") %></h4>
```

Comment:
@andy-armstrong @bjacobel Thanks for alerting me! Brian makes a good point - in general, we do need to update doc to reflect changed button labels - in procedure steps, as well as in screenshots - and Doc is not always aware of such changes if they're not done as part of some feature design work. But in this particular case I'd say go ahead - this seems to be a rare (only?) case remaining from what I remember as a previous effort by you, Andy, to standardize the verb to "Add" throughout most of the discussion area. Thanks!